### PR TITLE
PR main from contractor page

### PR DESCRIPTION
**Pull request by inikasingh21**
Added contractor page and 3 widgets for the layout of the page
image
image

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -11,12 +13,13 @@ class MyApp extends StatelessWidget {
...
-      title: 'Flutter Demo',
+      title: 'Contractor Webapp',
       theme: ThemeData(
-
+        scaffoldBackgroundColor: const Color.fromARGB(255, 231, 228, 245),
         useMaterial3: true,
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const ResultsPage(), 
+      //home: const ContractorPage(),
```
Comment (erickjtorres) [inline lib/main.dart]: remove this comment please

File: lib/widgets/c_card_widget.dart
Comment (erickjtorres) [inline lib/widgets/c_card_widget.dart:1]: Can you update the name of this. Should match the name of the class (ie: contractor_card.dart)

File: lib/results_page.dart [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    rating: 4.7,
+    tags: ["HVAC", "Air Conditioning", "Heating"],
+    phone: '', email: '',
+    aboutUs: 'blah blah blah',
+  ),
+];
+
+
+class ResultsPage extends StatelessWidget{  //defined class that extends statefulwidget (means app changes over time)
+  const ResultsPage({super.key}); //constructor for creating the class (when creating a new object from the class the cunstroctor is automatically called to initialize it in which the constructor is passing information (like the key) when creating widgets)
```
Comment (erickjtorres) [inline lib/results_page.dart]: nit. Please remove these comments

File: lib/results_page.dart [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+         ],)
+      ],
+    ),
+  );
+}
+}
+
+//creating a class for the tags
+//this can be added later: this widget accepts text and color, in which depending on the tag, the color of the oval should be different
+class OvalTags extends StatelessWidget {
```
Comment (erickjtorres) [inline lib/results_page.dart]: did we not move this to another page already? Might just need to merge the other branch and these changes will update

**Review by erickjtorres — APPROVED** [review]

**Review by erickjtorres — CHANGES_REQUESTED** [review]
just some small changes requested. Nice work!

**Review by erickjtorres — APPROVED** [review]

**Review by bridgettorres — APPROVED** [review]

**Pull request merged**